CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+### citus v11.3.1 (February 12, 2024) ###
+
+* Fixes a bug related to non-existent objects in DDL commands (#6984)
+
+* Fixes a bug that could cause COPY logic to skip data in case of OOM (#7152)
+
+* Fixes a bug with deleting colocation groups (#6929)
+
+* Fixes incorrect results on fetching scrollable with hold cursors (#7014)
+
+* Fixes memory and memory context leaks in Foreign Constraint Graphs (#7236)
+
+* Fixes replicate reference tables task fail when user is superuser (#6930)
+
+* Fixes the incorrect column count after ALTER TABLE (#7379)
+
+* Improves citus_shard_sizes performance (#7051)
```

Comment:
let's refer to PRs merged into main branch instead of their release branch variants

```suggestion
* Improves citus_shard_sizes performance (#7050)
```

Reply:
Fixed